code-challenges.test.js

```diff
@@ -33,21 +74,65 @@ const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
-
+//pseudocode:
+// Create a function named evenlyDivByThree
+//input: object 1, object 2, object 3
+//output: whether number value in objects are evenly divisible by three, meaning no remainder.
+//will need 3 return statements
+// conditionals to do the math and compare to input values.
+//return string that states if it is evenly divisible by three.
+const evenlyDivByThree = () => {
```

Comment:
You are really close here.  Consider moving the logic from your test into the conditional statements.  You will also need to define a parameter and use dot notation to access the value assigned to the key number.  Ex: object.number